livre/eq/Channel.cpp

```diff
@@ -291,6 +292,7 @@ struct Channel::Impl
         renderPipeline.render( pipe->getFrameData()->getVRParameters(),
                                _frameInfo,
                                {{ _drawRange.start, _drawRange.end }},
+                               pipe->getFrameData()->getVolumeSettings().getDataSourceRange(),
```

Comment:
I think I already mentioned in other code review that this function has too many parameters. Can you create a struct to pass them.